From cecc138ad15fd81358bb614a1545fd1392509814 Mon Sep 17 00:00:00 2001
From: Klaus Zimmermann <klaus.zimmermann@quansight.com>
Date: Fri, 22 Mar 2024 09:32:39 +0100
Subject: [PATCH] Remove QUIET from cmake rocm detection

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 90743ab..6e2a893 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ set(hipcc_NAME "${PROJECT_NAME}")
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
-find_package(ROCM QUIET)
+find_package(ROCM)
 if(ROCM_FOUND)
   include(ROCMSetupVersion)
   rocm_setup_version(VERSION "${hipcc_VERSION}")
-- 
2.40.1

